invokeai/frontend/web/src/features/parameters/components/Upscale/ParamSimpleUpscale.tsx

```diff
@@ -0,0 +1,46 @@
+import { Combobox, FormControl, FormLabel } from '@invoke-ai/ui-library';
+import { useAppDispatch, useAppSelector } from 'app/store/storeHooks';
+import { useModelCombobox } from 'common/hooks/useModelCombobox';
+import { simpleUpscaleModelChanged } from 'features/parameters/store/upscaleSlice';
+import { memo, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSpandrelImageToImageModels } from 'services/api/hooks/modelsByType';
+import type { SpandrelImageToImageModelConfig } from 'services/api/types';
+
+const ParamSimpleUpscale = () => {
```

Comment:
I would recommend reusing `ParamSpandrelModel`. You could add a prop for which part of the uspcale state it should be using/updating